### PR TITLE
PT-1140 Split terms of Use link in footer

### PR DESCRIPTION
**Pull request by franck-fourel**

### CODE REVIEW

File: app/components/footerlinks/footerlinks.js [hunk 1]
```diff
@@ -77,13 +78,27 @@ const FooterLinks = (props) => {
           onClick={metricFnMkr('Support')}
           target="_blank">{t(SupportText)}</a>
     </div>
-    <div className='footer-link'>       
+    {__BRANDING__ === 'diabeloop' ? (
+       <div className='footer-link'>
```
Comment (yvon-dblg) [inline app/components/footerlinks/footerlinks.js]:
```suggestion
       <div className='footer-link'>
```

File: app/components/footerlinks/footerlinks.js [hunk 2]
```diff
@@ -77,13 +78,27 @@ const FooterLinks = (props) => {
...
+          id='privacy'
+          onClick={metricFnMkr('Privacy')}
+          target='_blank'>{t('Privacy Policy')}</a> 
+        &nbsp;&&nbsp;
+        <a
+          href={urlLegal}
+          id='terms'
+          onClick={metricFnMkr('TermsOfUSe')}
+          target='_blank'>{t('Tidepool Applications Terms of Use')}</a>
+    </div> ) : (    <div className='footer-link'>
```
Comment (yvon-dblg) [inline app/components/footerlinks/footerlinks.js]:
```suggestion
    </div> ) : (
      <div className='footer-link'>
```

File: app/components/footerlinks/footerlinks.js [hunk 3]
```diff
@@ -77,13 +78,27 @@ const FooterLinks = (props) => {
...
     </div>
-    <div className='footer-link'>       
+    {__BRANDING__ === 'diabeloop' ? (
+       <div className='footer-link'>       
+        <a
+          href={urlPrivacy}
+          id='privacy'
+          onClick={metricFnMkr('Privacy')}
+          target='_blank'>{t('Privacy Policy')}</a> 
+        &nbsp;&&nbsp;
```
Comment (yvon-dblg) [inline app/components/footerlinks/footerlinks.js]:
```suggestion
```
React ignore this

File: app/components/footerlinks/footerlinks.js [hunk 4]
```diff
@@ -77,13 +78,27 @@ const FooterLinks = (props) => {
...
+          onClick={metricFnMkr('TermsOfUSe')}
+          target='_blank'>{t('Tidepool Applications Terms of Use')}</a>
+    </div> ) : (    <div className='footer-link'>       
         <a
           href={urlLegal}
           id='legal'
           onClick={metricFnMkr('PP and TOU')}
           target='_blank'>{t('Privacy and Terms of Use')}</a>
-    </div>
+    </div>)
```
Comment (yvon-dblg) [inline app/components/footerlinks/footerlinks.js]:
```suggestion
      </div>)
```

File: app/components/footerlinks/footerlinks.js (hunk 3 above)
Comment (franck-fourel) [reply app/components/footerlinks/footerlinks.js]: Yep it's ignored by React but work as expected

**Pull request merged**